release/release_tests.yaml

```diff
@@ -3413,7 +3413,7 @@
 
   run:
     timeout: 1800
-    script: OMP_NUM_THREADS=64 RAY_ADDRESS= python run_microbenchmark.py
+    script: OMP_NUM_THREADS=64 RAY_ADDRESS=auto python run_microbenchmark.py
```

Comment:
Any ideas why this worked before?

Reply:
probably because of this pr https://github.com/ray-project/ray/commit/55a0f7bb2db941d8c6ff93f55e4b3193f404ddf0